sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -789,6 +797,39 @@ case class Range(
     }
   }
 
+  private def computeHistogramStatistics() = {
```

Comment:
Trivial: you could put a return type here.
Is there any concern with the overhead of computing the histogram? It only happens when enabled of course. I just wonder if we need to micro-optimize the loop below with something more performant if this is critical, but maybe it isn't.

Reply:
> Trivial: you could put a return type here.Yes, I added the return type

Yes, I updated with the return type.

> Is there any concern with the overhead of computing the histogram? It only happens when enabled of course. I just wonder if we need to micro-optimize the loop below with something more performant if this is critical, but maybe it isn't.

This is not a performance intensive path. This will execute during planning phase. Also the loop size is numBins, which is 254 by default. So I am not sure we need to micro-optimise the loop?